src/System.Management.Automation/engine/hostifaces/LocalConnection.cs

```diff
@@ -678,30 +678,27 @@ private void DoOpenHelper()
             s_runspaceInitTracer.WriteLine("runspace opened successfully");
 
             // Now do initial state configuration that requires an active runspace
-            if (InitialSessionState != null)
```

Comment:
Do we know that InitialSessionState is always non-null now?  Should we add an assert here?

Reply:
Yes, it's always not null now. There is an assert at the beginning of `DoOpenHelper()`.